app/Tickets/Flipside/class.FlipsideTicketRequestEmail.php

```diff
@@ -41,7 +41,7 @@ public function getHTMLBody()
                 Reminder:  The dates for mailing your ticket request envelope are '.$this->settings['mail_start_date'].' through '
                 .$this->settings['request_stop_date'].'. Your postmark must be within that date range.<br><br>
                 If you need to update your ticket registration before mailing in your ticket request, return to 
-                <a href="https://secure.burningflipside.com/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
+                <a href="'.$this->secureUrl.'/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
```

Comment:
i guess i'm not clear on what you're labeling as being excessive coupling.. 
if it's the mailer accessing our settings.. pull #19 still has that going on..
if it's mail copy being hard coded into a class.. i completely agree..